### PR TITLE
5341: Fixes upload bug

### PR DESCRIPTION
**Pull request by meltheadorable**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -48,7 +48,8 @@ gem "active_model_serializers", "~> 0.9.3"
 gem "api-pagination", "~> 4.1.1"
 
 # Auto rank maintenance for sorted lists.
-gem "acts_as_list", github: "swanandp/acts_as_list", branch: "master"
+# The former master branch we were relying on is now a stable release
+gem "acts_as_list", "~> 0.8.0"
```
Comment (hooverlunch) [inline Gemfile]: Nice

File: Gemfile.lock [hunk 2]
```diff
@@ -9,20 +9,12 @@ GIT
       railties
 
 GIT
-  remote: git://github.com/sassafrastech/rack-attack.git
+  remote: https://github.com/sassafrastech/rack-attack.git
```
Comment (hooverlunch) [inline Gemfile.lock]: Nice tidy.

File: spec/models/user_batch_spec.rb [hunk 3]
```diff
@@ -51,6 +51,11 @@
     expect(2).to eq(ub.users.size)
   end
 
+  it "succeeds when headers have trailing invisible blanks" do
+    ub = create_user_batch("abnormal_headers.xlsx")
+    expect(ub).to be_succeeded
+  end
```
Comment (hooverlunch) [inline spec/models/user_batch_spec.rb]: Lovely.

**Comment by rgalusca** [comment]
I cannot merge this as I do not have write permission. Can someone give me permission? Thanks!

**Pull request merged**